src/mobile/components/ModalWithdrawConfirmation/index.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { Button } from 'react-bootstrap';
+import { useIntl } from 'react-intl';
+import { Decimal } from '../../../components';
+import { Modal } from '../../components';
+
+interface ModalWithdrawConfirmationProps {
+    amount: string;
+    currency: string;
+    onSubmit: () => void;
+    onDismiss: () => void;
+    rid: string;
+    show: boolean;
+    precision: number;
+}
+
+const ModalWithdraw = (props: ModalWithdrawConfirmationProps) => {
+    const intl = useIntl();
+
+    const translate = (e: string) => {
```

Comment:
why do we need this function?
let's use `formatMessage`

`const { formatMessage } = useIntl();`

and use it everywhere instead